unit/src/ElementFragmentAlgorithmTest.C

```diff
@@ -146,8 +146,6 @@ TEST(ElementFragmentAlgorithm, test1b)
 
   MyMesh.updateTopology();
 
-  //  MyMesh.printMesh();
```

Comment:
Would you mind converting these commented out versions to:
```
#ifndef NDEBUG
  MyMesh.printMesh();
#endif

Reply:
?!